client.go

```diff
@@ -70,12 +70,12 @@ type Auth struct {
 }
 
 // NewNativeClient creates a new Client using the golang ssh library
-func NewNativeClient(user, host, clientVersion string, port int, auth *Auth) (Client, error) {
+func NewNativeClient(user, host, clientVersion string, port int, auth *Auth, hostKeyCallback ssh.HostKeyCallback) (Client, error) {
```

Comment:
I'm not a fan of changing exported function signatures, but am open to suggestions, and feel maybe i could release a v0.0.0 or something then update this for a new major version.